go.mod

```diff
@@ -5,8 +5,10 @@ go 1.15
 require (
 	github.com/alecthomas/kong v0.2.11
 	github.com/apex/log v1.9.0
+	github.com/itchio/go-brotli v0.0.0-20190702114328-3f28d645a45c
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/satori/go.uuid v1.2.0
 	github.com/stretchr/testify v1.6.1
 	gopkg.in/yaml.v2 v2.3.0
+	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
 )
```

Comment:
Could you see whether we could switch our yaml.v2 usage to yaml.v3, so that we'd only include that in go.mod going forward?

Reply:
Strange, yaml.v3 fails with Unmarshal, even though its signature and documentation is identical.